documentation/04_tutorials/10-part-ix-ui-and-combat.html.md

```diff
@@ -134,7 +134,7 @@ This all seems simple enough, but it's actually going to require several compone
 		FlxG.collide(_player, _mWalls);
 		FlxG.overlap(_player, _grpCoins, playerTouchCoin);
 		FlxG.collide(_grpEnemies, _mWalls);
-		checkEnemyVision();
+		checkEnemyVision;
```

Comment:
It's not a function call without the braces, so it won't do anything.

http://try.haxe.org/#C15AC

What error were you getting?